lib/y2_installbase.pm

```diff
@@ -183,9 +183,12 @@ highlight cursor one item down.
 =cut
 
 sub move_down {
+    # Sending 'down' twice, followed by 'up', scrolls to the intended item.
+    wait_screen_change { send_key 'down' };
     my $ret = wait_screen_change { send_key 'down' };
-    workaround_bsc1189550() if (!$workaround_bsc1189550_done && is_sle('>=15-sp3'));
     last if (!$ret);    # down didn't change the screen, so exit here
+    wait_screen_change { send_key 'up' };
+    workaround_bsc1189550() if (!$workaround_bsc1189550_done && is_sle('>=15-sp3'));
```

Comment:
we should not do it for each step, it might cause issues in slower architectures:
we should try to see if previous workaround in single place still works, see previous commit:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13155/files
otherwise find a way to do the trick after certain number of items.

Reply:
You are doing the same steps no? after every move down, you move one more down and up

Reply:
yes, the difference is that is encapsulated in the function that only does it once at the beginning, with the first down.

Reply:
if we don't find anything better and this change needs to be merge to unblock further testing, I would suggest to change line 191 removing variable `$workaround_bsc1189550_done` and replacing the workaround.

Reply:
without video I cannot tell, but it looks like the current workaround might not work anymore because the alt-end and alt-home are lost when they are sent if the selection is on a category:
https://openqa.suse.de/tests/9806816#step/select_patterns/11 (just a theory)
because this up-down trick should be equivalent to that.

Reply:
I am a bit confused on what to do, your second comment now says something else, but anyway, I think I can mark this as resolved, I deleted the old workaround completely and it should not run for older products.